librz/include/rz_types.h

```diff
@@ -151,6 +151,10 @@ typedef enum {
 #define __KFBSD__ 0
 #endif
 
+#ifdef __MSYS__
+#define __UNIX__ 1
```

Comment:
Is this really the way to do it? This will bring the `__UNIX__` define to all projects that use rizin api.

Reply:
1. It probably should be done in the future by the Meson
2. I just mirror logic of the other platforms, e.g. HAIKU - see the surrounding lines.

Reply:
I probably wouldn't do it in meson either but use a custom unix define, otherwise the headers outside of meson will be broken again. But considering 2, lgtm for now.